java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java

```diff
@@ -90,6 +95,23 @@ protected void doBuild(Path directory, Map<String, String> metadata) throws Buil
   }
 
   /*
+   * Deletes a directory and all of its contents.
+   */
+  private void deleteDirectoryRecursive(Path path) throws IOException {
```

Comment:
I could, but I did just remove the Apache Commons IO dependency. Do you think it's worth re-adding that dependency for just this one method?